img-buildkite-64.json

```
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "oxc",
+    "image": "img-buildkite-64/oxc.png",
+    "category": "Buildkite",
+    "aliases": ["oxc-rs", "oxlint"]
```

Nit. Can we have this alias ordering consistent with the README entry?

```suggestion
    "aliases": ["oxlint", "oxc-rs"]
```

Or have them both the other way around, you call.